package.json

```diff
@@ -16,43 +16,44 @@
   "bugs": "https://github.com/DockYard/ember-composable-helpers/issues",
   "homepage": "https://github.com/DockYard/ember-composable-helpers",
   "engines": {
-    "node": ">= 0.10.0"
+    "node": ">= 4"
   },
-  "author": [
+  "authors": [
```

Comment:
@cibernox Is this the typo you mentioned in the PR description? NPM's `package.json` only supports a single [author](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) (but can have multiple _collaborators_). You would have to add an `AUTHORS.md` file at the root of the project. An alternative would be to set the author as `DockYard <oss@dockyard.com>` and list these existing entries under `collaborators`. That could probably be separated from this PR though.